styleguide/source/_patterns/03-organisms/four-up-teaser.twig

```diff
@@ -1,6 +1,6 @@
 <div class="ama__four-up-teaser">
-  {% for item in fourUpTeaser.items[0:4] %}
```

Comment:
This shouldn't be removed. It's an important note that the items never exceed 4 even when more than 4 are added.